github-dark.css

```diff
@@ -3016,7 +3017,8 @@
   .discussion-item-private, .label-select-menu div.labelstyle-000000.selected,
   .marketplace-plan-emphasis, .header-nav-link, .header-navlink,
   .Header .header-search-scope, a.notification-indicator, .HeaderNavlink,
-  .UnderlineNav-item:hover, .UnderlineNav-item:focus, .project-pane-close:hover {
+  .UnderlineNav-item:hover, .UnderlineNav-item:focus, .project-pane-close:hover,
+  .octotree_help > span:before {
```

Comment:
There's already a `.octotree_help > span` earlier in this rule, that's where the `:before` should go instead of adding a new selector